common/vpcclient/instances/volume_mounter_service.go

```diff
@@ -47,10 +48,9 @@ type VolumeAttachService struct {
 
 // IKSVolumeAttachService ...
 type IKSVolumeAttachService struct {
-	client                  client.SessionClient
-	pathPrefix              string
-	receiverError           error
-	populateQueryParameters func(request *client.Request, volumeAttachmentTemplate *models.VolumeAttachment) *client.Request
+	client        client.SessionClient
+	pathPrefix    string
+	receiverError error
```

Comment:
why `populateQueryParameters` is removed?

Reply:
Because when populateQueryParameters is not used. When we are forming IKSVolumeAttachService we are not passing populateQueryParameters anywhere.